index.html

```diff
@@ -19,19 +19,33 @@
 </head>
 
 <body>
-    <div class="X">
+
+    <ul class="topnav">
+        <li><a href="#home">Home</a></li>
```

Comment:
I noticed that your links don't actually "jump" to the appropriate section of your web page. I think you need to add `id` tags to your various `div`s with the appropriate names (ie: "Bio") in order to allow the links to work